### PR TITLE
Add pages

### PR DESCRIPTION
**Pull request by qwibbler**
Added pages (home, calculate, quote) and styling.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
+// import CalculatorPage from './pages/CalculatorPage';
 import reportWebVitals from './reportWebVitals';
```
Comment (DanSam5K) [inline src/index.js]: - [x] It would really be great if you can take out all unused code, what do you think? 🤔

**Review by DanSam5K — CHANGES_REQUESTED** [review]
## Hi @qwibbler, ♻️ Required Changes ♻️
### Highlights 💥💥 
- [x] Your project is excellent and amazing ✔️ 
- [x] Excellent sense of choice with the logo and nice user interface ✔️ 
- [x] Calculator works perfectly as expected and all linters check passed ✔️  
 
Good job so far!
It was actually very difficult not approving this project after I noticed some unused code 😢 
But you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (qwibbler) [reply src/index.js]: I checked everything else! Oh well, I'll do that right now. Thanks! 😅

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @qwibbler, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**